lein-jank/test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
Please use `#!/usr/bin/env bash`, which makes it more portable. Those of us on NixOS appreciate it. :joy: